platform_umbrella/apps/common_ui/lib/common_ui/components/alert.ex

```diff
@@ -86,29 +85,46 @@ defmodule CommonUI.Components.Alert do
     )
   end
 
-  defp type_class("inline"), do: "px-3.5 py-2.5"
-
-  defp type_class("fixed") do
-    "fixed shadow-lg z-50 max-w-96 px-5 py-4 shadow-xl shadow-gray-darkest/10 z-50 hidden"
+  defp variant_class("info", type) do
+    [
+      "text-sky-500 dark:text-white",
+      type != "minimal" && "bg-sky-50 dark:bg-sky-500 border border-sky-100 dark:border-none",
```

Comment:
This pattern of `bg-sky-50 dark:bg-sky-500 dark:border-none` being added if the type is not minimal can be extracted to repeat yourself less.